pkg/ngrokapidriver/driver.go

```diff
@@ -183,28 +191,24 @@ func (nc ngrokAPIDriver) GetReservedDomains(ctx context.Context, edgeID string)
 	}
 	hostPortDomains := []string{}
 	for _, hostport := range *edge.Hostports {
-		hostPortDomains = append(hostPortDomains, strings.Split(hostport, ":")[0])
+		domain, _, err := net.SplitHostPort(hostport)
```

Comment:
I mean `net.SplitHostPort` exists so I guess we should use it why not. :man_shrugging: